sdk/core/azure-core-tracing-opentelemetry/src/main/java/com/azure/core/tracing/opentelemetry/implementation/OpenTelemetrySpanSuppressionHelper.java

```diff
@@ -1,99 +0,0 @@
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License.
-
-package com.azure.core.tracing.opentelemetry.implementation;
```

Comment:
it's not needed anymore. This code was interacting with OTel Java agent, but now the agent contains Azure SDK instrumentation (thanks to Trask 🥳) and we don't need the suppression here. 

This code was also noop because of OTel instrumentation API changes.